app/qml/map/RecordingToolbar.qml

```diff
@@ -104,7 +104,6 @@ Item {
         Item {
             Layout.fillWidth: true
             height: parent.height
-            visible: root.pointLayerSelected ? false : true
```

Comment:
I'd prefer keeping remove button hidden for simple point geometries. It opens another level of possible bugs if we would show it here.